ShellPkg/Library/UefiShellDebug1CommandsLib/MemMap.c

```diff
@@ -210,7 +210,14 @@ ShellCommandRunMemMap (
       if (Status == EFI_BUFFER_TOO_SMALL) {
         Size       += SIZE_1KB;
         Descriptors = AllocateZeroPool (Size);
-        Status      = gBS->GetMemoryMap (&Size, Descriptors, &MapKey, &ItemSize, &Version);
+        // MU_CHANGE [BEGIN] - CodeQL change
+        if (Descriptors == NULL) {
+          ShellPrintHiiEx (-1, -1, NULL, STRING_TOKEN (STR_GEN_OUT_MEM), gShellDebug1HiiHandle, L"memmap");
+          ShellStatus = SHELL_OUT_OF_RESOURCES;

```

Comment:
Do we need to return here or does gBS->GetMemoryMap handle NULL descriptors?

Reply:
It does. It returns `EFI_INVALID_PARAMETER`. That will print the get memory map failed message.